### DISCUSSION TITLE
Welcome to homebridge-bond Discussions!

### DISCUSSION THREAD

**Discussion by aarons22** [aarons22/homebridge-bond#123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aoh2306** [comment]
Hi I'm Arthur and I am interested in the Bond Homebridge community given my interest in the Minka-Aire fans. I would like to learn how to control the fans using Apple HomeKit. I understand thus far that Homebridge is a firmware solution that allows HomeKit to communicate with the fans via the Bond application. Any additional information on this structure, function and access to the Homebridge software would b greatly appreciated. Thanks in advance.
